aws/resource_aws_emr_cluster.go

```diff
@@ -1741,3 +1962,272 @@ func fetchAllEMRInstanceGroups(conn *emr.EMR, clusterID string) ([]*emr.Instance
 
 	return groups, err
 }
+
+func readInstanceFleetConfig(data map[string]interface{}) *emr.InstanceFleetConfig {
+
+	config := &emr.InstanceFleetConfig{
+		InstanceFleetType:      aws.String(data["instance_fleet_type"].(string)),
```

Comment:
As part of removing the configuration argument, this can be passed into the function depending on whether it is the `master_instance_fleet` or `core_instance_fleet`. 👍